admin-ui/src/components/add-account-dialog.tsx

```
@@ -0,0 +1,496 @@
+              <Button variant="outline" onClick={handleCancel}>
+                {t("accountManagement.cancel")}
+              </Button>
+            </DialogFooter>
+          </>
+        )}
+
+        {step === "success" && (
+          <>
+            <DialogHeader>
+              <DialogTitle>
+                {reauthAccountId
```

ESLint

[error] Insert `·?`

(prettier/prettier)